pkg/hive/resources.go

```diff
@@ -10,16 +10,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// Changing values of these constants most likely would require
+// some sort of migration on the Hive cluster for existing clusters.
 const (
-	ClusterDeploymentName      = "cluster"
-	kubesecretName             = "admin-kube-secret"
-	servicePrincipalSecretname = "serviceprincipal-secret"
+	ClusterDeploymentName             = "cluster"
+	aroServiceKubeconfigSecretName    = "aro-service-kubeconfig-secret"
+	clusterServicePrincipalSecretName = "cluster-service-principal-secret"
 )
 
-func kubeAdminSecret(namespace string, kubeConfig []byte) *corev1.Secret {
```

Comment:
Renamed to `aroServiceKubeconfigSecret` to avoid confusion. While from Hive point of view it is an admin kubeconfig, from ARO service perspective - we should not be using admin kube config (intened to be used by the end customer) and use ARO service kubeconfig.